frontend/src/routes/GlobalRoutes.tsx

```diff
@@ -1,25 +1,16 @@
-import React, {useEffect, useState} from 'react';
+import React from 'react';
 import Header from "../components/header/Header";
 import RouteMain from "./RouteMain";
 import RouteAuth from "./RouteAuth";
 import {useAppSelector} from "../stores/redux/hooks";
-import {selectLoggedUser} from "../stores/redux/loggedUser/loggedUserSlice";
 import _ from "lodash";
 import Notification from "../components/notification/Notification";
 import Footer from "../components/footer/Footer";
+import {RootState} from "../stores/redux/store";
 
 const GlobalRoutes: React.FC = () => {
 
-    const [isLogged, setIsLogged] = useState<boolean>(false);
-    const loggedUser = useAppSelector(selectLoggedUser);
-
-    const updateIsLogged = () => {
-        setIsLogged(!_.isEmpty(loggedUser.token));
-    }
-
-    useEffect(() => {
-        updateIsLogged();
-    }, [loggedUser]);
+    const isLogged = useAppSelector((state: RootState) => !_.isEmpty(state.loggedUserState.token));
```

Comment:
Maybe it makes sense to move this method to slice to reuse it somewhere else in the future